util/environment/environment.go

```diff
@@ -1,33 +1,39 @@
 package environment
 
 import (
+	"os"
 	"sort"
 	"strings"
 )
 
-// MergeEnvironments merges env2 into env1, overwriting any existing variable
-// in env1, except for the "special" variables PATH, CPATH and LD_LIBRARY_PATH.
+var pathSeparator = string(os.PathListSeparator)
+
+// MergeEnvironments merges one or more sets of environment variables,
+// overwriting any existing variable in the preceding set, except for the
+// "special" variables PATH, CPATH and LD_LIBRARY_PATH.
 //
 // The "special" variables PATH, CPATH and LD_LIBRARY_PATH are merged by
 // prepending the value from env2 to the value in env1, effectively giving
 // priority to the value from env2.
 //
 // The expected format for an environment variable definition is VAR=VALUE. Any
 // malformed environment variable definition will be discarded by the merge.
-func MergeEnvironments(env1, env2 []string) []string {
-	e1 := toMap(env1)
-	e2 := toMap(env2)
+func MergeEnvironments(ea []string, es ...[]string) []string {
```

Comment:
We need to rephrase the comment above `MergeEnvironments()` to explain how it merges the arguments "right to left" in succession, instead of just `env2` into `env1`.